pkg/adapter/mtping/runner.go

```diff
@@ -71,7 +72,10 @@ func NewCronJobsRunner(ceClient cloudevents.Client, kubeClient kubernetes.Interf
 }
 
 func (a *cronJobsRunner) AddSchedule(source *v1beta2.PingSource) cron.EntryID {
-	event := makeEvent(source)
+	event, err := makeEvent(source)
+	if err != nil {
+		a.Logger.Error("failed to makeEvent: ", err)
```

Comment:
a.Logger.Error("failed to makeEvent: ", zap.Error(err))

Reply:
thanks! done.